converter/src/test/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoderTest.java

```diff
@@ -141,15 +141,9 @@ public void testInternalEncode() throws EncodeException {
 
 		Map<?, ?> clinicalDocMap = ((Map<?, ?>) testJsonWrapper.getObject());
 
-		assertWithMessage("Must have a correct program name")
```

Comment:
I would recommend having a test that ensures the `ClinicalDocumentEncoder` does _not_ encode the program nor the entity ID.